app/src/main/kotlin/cz/covid19cz/erouska/ui/main/MainActivity.kt

```diff
@@ -33,6 +33,13 @@ import javax.inject.Inject
 class MainActivity :
     BaseActivity<ActivityMainBinding, MainVM>(R.layout.activity_main, MainVM::class) {
 
+    companion object {
```

Comment:
I would move this to Analytics class

Reply:
it's better to let the consumer control what events are sent. but they may be kept in one place, especially in a small-sized app such as this one, if you prefer.